### PR TITLE
Add the possibility to configure toast

### PR DESCRIPTION
**Pull request by fjaguado**
# PR Details

Now is possible to configure some Toast behavior when SystelabComponentModule is imported using the forRoot method and passing to it a configuration object.

What we can configure: 
- Toast fixed size between small and large
- Enable auto width
- Show close button 
- Change timeout value

## Related Issue

#686 
#744 

## Motivation and Context

We need to improve the speed of E2E when we have to test components with Toast because of the 5 seconds dismiss timeout. 
In other way some projects have the need to print a larger toast.

## How Has This Been Tested
 
Component tests included in the Showcase application

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ x ] I have read the **CONTRIBUTING** document
- [ x ] My code follows the code style of this project
- [ x ] My change requires a change to the documentation 
- [ x ] I have updated the documentation accordingly (README.md for each UI component)
- [ ] I have added tests to cover my changes (at least 1 spec for each UI component with the same coverage as the master branch)
- [ x ] All new and existing tests passed
- [ x ] A new branch needs to be created from master to evolve previous versions
- [ x ] Increase version in package.json following [Semantic Versioning](https://semver.org/)
- [ x ] All UI components must be added into the showcase (at least 1 component with the default settings)
- [ x ] Add the issue into the right [project](https://github.com/systelab/systelab-components/projects) with the proper status (In progress)

### CODE REVIEW

File: projects/showcase/src/app/components/toast/showcase-toast.component.ts [hunk 1]
```diff
@@ -19,10 +22,42 @@ export class ShowcaseToastComponent {
...
+		})
+		this.toastService.showSuccess('This is a successful large action toast');
+	}
+
+	public showSuccessAutoWidthToast(): void {
+		this.toastService.setConfig({
+			...this._defaultConfig,
+			autoWidth: true,
+		})
+		this.toastService.showSuccess('This is a successful auto width action toast. It grows according on its content');
```
Comment (Franek75) [inline projects/showcase/src/app/components/toast/showcase-toast.component.ts:45]: according to

File: projects/systelab-components/README.md [hunk 2]
```diff
@@ -63,8 +63,38 @@ NgModule({
...
+    toast: {
+        autoWidth: false,
+      fixedSize: ToastSize.small,
+      showCloseButton: false,
+      timeout: 5000,
+    }
+}
+```
 
 If you need to import SystelabComonentsModule in a lazy loaded module you must use it without call the *forRoot* method again. If you call twice or more, you will
```
Comment (Franek75) [inline projects/systelab-components/README.md:99]: typo (Comonents)

File: projects/systelab-components/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components",
-  "version": "17.0.1",
+  "version": "17.2.0",
```
Comment (OscarDeDios) [inline projects/systelab-components/package.json:3]: why 7.2? should be 17.1.0

**Review by jserra91 — APPROVED** [review]

**Review by rochiebcn — APPROVED** [review]
Thank you!

**Review by OscarDeDios — APPROVED** [review]

**Pull request merged**